website/docs/configuration/anonymous-access.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Anonymous Access
+---
+
+:::danger Important
+Alone, this is an **insecure** method of securing your dashboard.
+
+It is designed to be used with other external authentication systems like auth proxies.
```

Comment:
I'm not sure this bit is needed, it does nothing to support them.

Reply:
Yes, rephrase?

> If you decide to use anonymous access, you should be cautious about exposing the endpoint and consider implementing additional authentication measures such as an auth proxy.

Reply:
Sounds better 👍🏻